Makefile

```diff
@@ -22,13 +25,22 @@ quality-js:
 
 deps:
 	pip install -r requirements.txt
-	bower install
 
 deps-test:
 	pip install -r test_requirements.txt
+	bower install
 
 tools:
-	npm install "eslint@^2.12.0" eslint-config-edx "eslint-plugin-dollar-sign@0.0.5" "eslint-plugin-import@^1.9.2"
+	npm install bower "eslint@^2.12.0" eslint-config-edx "eslint-plugin-dollar-sign@0.0.5" "eslint-plugin-import@^1.9.2"
+
+send_coveralls:
+	coveralls-lcov -v -n video_xblock/static/coverage/PhantomJS\ 2.1.1\ \(Linux\ 0.0.0\)/lcov.info > coverage.json
+	coveralls --merge=coverage.json
+
+prepare-for-js:
+	export DISPLAY=:99.0
+	sh -e /etc/init.d/xvfb start
```

Comment:
Move these two lines to `test-js`
Probably you also need to add `xvfb stop` after tests have ran